quilt/tools/command.py

```diff
@@ -581,6 +587,7 @@ def main():
     build_p.add_argument("package", type=str, help="Owner/Package Name")
     buildpath_group = build_p.add_mutually_exclusive_group(required=True)
     buildpath_group.add_argument("-d", "--directory", type=str, help="Source file directory")
```

Comment:
I suggested -d <directory> and -g (no arg, generate only). Or, we could make generate (without building) a separate command, quilt generate. Generating the build file actually doesn't need a username/package so maybe that would be clearer.